src/Akeneo/Tool/Bundle/MessengerBundle/Command/PurgeMessengerCommand.php

```
@@ -66,13 +60,13 @@ protected function execute(InputInterface $input, OutputInterface $output): ?int
 
         try {
             $this->purgeDoctrineQueue->execute($tableName, $queueName, $olderThan);
-        } catch (DBALException $dbalException) {
-            $output->writeln($dbalException->getMessage());
+        } catch (\Exception $exception) {
```

Replacement is \Doctrine\DBAL\Exception but I am ok to catch all exception here